### PR TITLE
create store form

### PR DESCRIPTION
**Pull request by thucpv-1748**

### CODE REVIEW

**Comment by thucpv-1748** [comment]
https://edu-redmine.sun-asterisk.vn/issues/9187

File: app/Http/Controllers/Backend/StoreController.php [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            return redirect()->back()->with('error', $e->getMessage());
+        }
+    }
+
+    /**
+     * @param $id
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\Http\RedirectResponse|\Illuminate\View\View
+     */
+    public function editStore($id)
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/StoreController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/StoreController.php</br>**Line**: 84</br>Line indented incorrectly; expected 8 spaces, found 9

File: app/Http/Controllers/Backend/StoreController.php [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+         $store = $this->model->findOrFail($id);
+         if ($store) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/StoreController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/StoreController.php</br>**Line**: 86</br>Line indented incorrectly; expected 8 spaces, found 9

File: app/Http/Controllers/Backend/StoreController.php [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+             return view('layout/backend/storeform', compact('store'));
+         } else {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Backend/StoreController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.ScopeIndent.IncorrectExact</br>**Severity**: error</br>**File**: app/Http/Controllers/Backend/StoreController.php</br>**Line**: 88</br>Line indented incorrectly; expected 8 spaces, found 9

**Pull request merged**